src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -284,28 +76,22 @@ public void beforeFindBy(By by, WebElement element, WebDriver driver) {
 
   @Override
   public void beforeClickOn(WebElement element, WebDriver driver) {
-    logJSError(driver);
+    LOG.logJSError();
   }
 
   @Override
   public void afterClickOn(WebElement element, WebDriver driver) {
-    StringBuilder builder = new StringBuilder();
-    builder.append("<tr class=\"success lowLevelAction\"><td>click</td><td>" + lastFindBy
-                   + "</td><td> <br/> &nbsp;</td></tr>");
-    CommonUtils.appendTextToFile(logPath, builder.toString());
+    LOG.info("CLICK", lastFindBy.toString());
   }
 
   @Override
   public void afterChangeValueOf(WebElement element, WebDriver driver) {
-    StringBuilder builder = new StringBuilder();
-    builder.append("<tr class=\"success lowLevelAction\"><td>ChangeValueOfField</td><td>"
-                   + lastFindBy + "</td><td> <br/> &nbsp;</td></tr>");
-    CommonUtils.appendTextToFile(logPath, builder.toString());
+    LOG.info("CHANGE FIELD VALUE", lastFindBy.toString());
   }
 
   @Override
   public void onTestStart(ITestResult result) {
-    logsResults.clear();
+    LOG.clearLogs();
     String testName = result.getName().toString();
     String className = result.getTestClass().getName().toString();
     System.out.println(className + " " + testName);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)